assets/javascript/javascript.js

```diff
@@ -52,8 +54,10 @@ function checkLetters(letter) {
     for (var i = 0; i < numBlanks; i++) {
         if (wordIndex[i] === letter) {
             letterInWord = true;
+            break;
```

Comment:
Once you encounter a matching letter, then you probably want to quit looping through the word. the break statement exits the loop